vcr/vcr.go

```diff
@@ -550,5 +595,40 @@ func (c *vcr) Diagnostics() []core.DiagnosticResult {
 			Title:   "credential_count",
 			Outcome: credentialCount,
 		},
+		core.DiagnosticResultMap{
+			Title: "wallet_credential_count",
+			Items: c.wallet.Diagnostics(),
+		},
```

Comment:
So I know how to interpret these in the future: Should these credential counts contain the same value?

Reply:
?

Reply:
nvm, I thought I deleted this comment